src/05-invoicing/subscriptions.apib

```diff
@@ -139,3 +139,33 @@
                         + taxable (Money)
                         + tax (Money)
             + `web_url`: `https://focus.teamleader.eu/subscription_detail.php?id=e2314517-3cab-4aa9-8471-450e73449041` (string)
+
+### subscriptions.create [POST /subscriptions.create]
+
```

Comment:
Let's add a small description of the purpose of the endpoint. 
It may seem redundant or implicit but it's the difference between a nice formatted UI. 
e.g.: 

| With Description  | Without Description |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/9110098/165629731-ecf04a43-19cd-4bdb-acd3-6f94c687fa13.png)  | ![image](https://user-images.githubusercontent.com/9110098/165629761-b7646631-b2ee-465d-82a9-e519931b9f12.png)  |